src/ITripod.php

```diff
@@ -1,4 +1,11 @@
 <?php
+
+namespace Tripod;
```

Comment:
We've prefixed the namespace in another lib with `Talis` - do we want to do something similar here?
https://github.com/talis/persona-php-client/blob/master/src/Talis/Persona/Client/Login.php#L2

If we do it'll need an update to the composer.json file in order to work